cpp/include/IceUtil/ScannerConfig.h

```diff
@@ -7,21 +7,11 @@
 
 #include "Config.h" // Required by generated Scanners.
 
-//
```

Comment:
The actual fix is to include `<cstdint>` at the top of each Scanner.l. 

I did this in a previous PR for the Slice and IceGrid scanners.